scipy/special/add_newdocs.py

```diff
@@ -7097,17 +7097,47 @@ def add_newdoc(name, doc):
 
         (z)_m = z (z + 1) ... (z + m - 1)
 
+    See [dlmf]_ for more details.
+
     Parameters
     ----------
-    z : array_like
-        (int or float)
-    m : array_like
-        (int or float)
+    z, m : array_like
+        Real-valued arguments.
```

Comment:
Isn't `array_like` not quite right for the type field since you can have a scalar input for either `z`  or `m`?

Reply:
`array_like` includes scalar.